client/app/utils/numbers.js

```diff
@@ -18,4 +19,9 @@ const formatDistance = function formatDistance(distance, unit, precision = PRECI
 
 const formatPrice = (price, unit) => `${unit} ${price}`;
 
-export { formatDistance, formatPrice };
+const toFixedNumber = (number, limit, base) => {
```

Comment:
What does this function do? `toFixed` is standard JS, and this apparently does something else?